maestro-ios/src/main/java/ios/hierarchy/Frame.kt

```diff
@@ -0,0 +1,10 @@
+package ios.hierarchy
+
+import com.google.gson.annotations.SerializedName
+
+data class Frame(
+    @SerializedName("Width") val Width: Float,
```

Comment:
Since you are already using `SerializedName`, let's keep actual field names canonical (camelCase)

Reply:
Done, changed!